packages/contracts/src/dump.ts

```diff
@@ -0,0 +1,131 @@
+/* External Imports */
+import * as path from 'path'
+import { ethers } from 'ethers'
+import * as Ganache from 'ganache-core'
+import { deployAllContracts, RollupDeployConfig } from './deployment'
+
+interface StorageDump {
+  [key: string]: string
+}
+
+export interface StateDump {
+  contracts: {
+    ovmExecutionManager: string
+    ovmStateManager: string
+  }
+  accounts: {
+    [address: string]: {
+      balance: number
+      nonce: number
+      code: string
+      storage: StorageDump
+    }
+  }
+}
+
+const getChangedAccounts = async (cStateManager: any): Promise<string[]> => {
+  return new Promise<string[]>((resolve, reject) => {
+    const accounts: string[] = []
+    const stream = cStateManager._trie.createReadStream()
+
+    stream.on('data', (val: any) => {
+      accounts.push(val.key.toString('hex'))
+    })
+
+    stream.on('end', () => {
+      resolve(accounts)
+    })
+  })
+}
+
+const getStorageDump = async (
+  cStateManager: any,
+  address: string
+): Promise<StorageDump> => {
+  return new Promise<StorageDump>((resolve, reject) => {
+    cStateManager._getStorageTrie(address, (err: any, trie: any) => {
+      if (err) {
+        reject(err)
+      }
+
+      const storage: StorageDump = {}
+      const stream = trie.createReadStream()
+
+      stream.on('data', (val: any) => {
+        storage[val.key.toString('hex')] = val.value.toString('hex')
+      })
+
+      stream.on('end', () => {
+        resolve(storage)
+      })
+    })
+  })
+}
+
+export const makeStateDump = async (): Promise<any> => {
+  const ganache = Ganache.provider({
+    gasLimit: 100_000_000,
+    allowUnlimitedContractSize: true,
+    accounts: [
+      {
+        secretKey:
+          '0x29f3edee0ad3abf8e2699402e0e28cd6492c9be7eaab00d732a791c33552f797',
+        balance: 10000000000000000000000000000000000,
```

Comment:
nit: indentation off

Reply:
Oh rip, I think this is just `prettier` being funky.